README.md

```diff
@@ -257,25 +257,28 @@ be of the form: `foundryvtt-0.7.5.zip`
 
 | Name  | Purpose | Default |
 |-------|---------|---------|
-| `CONTAINER_PATCHES` | Set a path to a directory of shell scripts to be sourced after Foundry is installed but before it is started.  The path should be in `/data` or another persistent mount point in the container. e.g.; `/data/container_patches`  Patch files are sourced in lexicographic order.  `CONTAINER_PATCHES` are processed after `CONTAINER_PATCH_URLS`.| |
+| `CONTAINER_PATCHES` | Set a path to a directory of shell scripts to be sourced after Foundry is installed but before it is started.  The path should be in `/data` or another persistent mount point in the container. e.g.; `/data/container_patches`  Patch files are sourced in lexicographic order.  `CONTAINER_PATCHES` are processed after `CONTAINER_PATCH_URLS`.|  |
 | `CONTAINER_PATCH_URLS` | Set to a space-delimited list of URLs to be sourced after Foundry is installed but before it is started.  Patch URLs are sourced in the order specified.  `CONTAINER_PATCH_URLS` are processed before `CONTAINER_PATCHES`.  ⚠️ **Only use patch URLs from trusted sources!** | |
-| `CONTAINER_VERBOSE` | Set to `true` to enable verbose logging for the container utility scripts. | |
+| `CONTAINER_PRESERVE_CONFIG` | Normally new `options.json` and `admin.txt` files are generated by the container at startup.  Setting this to `true` preserves these files and prevents the container from managing them.  This means that changes to environment variables will not be applied to the server. | `false` |
```

Comment:
This is a great addition. I just noticed this recently, but I have so few custom options that it didn't affect me.